boilerplate

```diff
@@ -1,18 +1,4 @@
 <!DOCTYPE html>
 <html lang="en">
-    <head>
-        <meta charset="UTF-8" />
-        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-
-        <link rel="stylesheet" href="Index.css" />
-        <title>Title</title>
-    </head>
-    <body>
-
-        <header>
-            <h1>>Header</h1>
-        </header>
-
-        <script src="Index.js"></script>
```

Comment:
This is an example comment in a pull request: 

Please, when you have a moment, do a little research of ideal and state of the art usage of script tags (positioning in the html document) :)